website/content/docs/api-gateway/configuration/routes.mdx

```diff
@@ -78,6 +166,105 @@ The following example creates a route named `example-route` in namespace `gatewa
+| `add` | Configure this field to append the request header with a new value. It specifies the name of an HTTP header to append and the value(s) to add. You can specify the following configurations: <ul><li>`name`: Required string that specifies the name of the HTTP header to append.</li><li>`value`: Required string that specifies the value of the HTTP header to add.</li></ul> | List of objects | Optional |
+| `remove` | Configure this field to specifify an array of header names to remove from the request header. | Array of strings | Optional |
+
+
+### rules.filters.urlRewrite
+
+Specifies rules for rewriting the URL of incoming requests when `rules.filters.type` is configured to `URLRewrite`. 
+
+* Type: Object
+* Required: Optional
+
+### rules.filters.urlRewrite.path
+
+Specifies a list of objects that determine how Consul API Gateway rewrites URL paths (refer to [Reroute HTTP requests](/docs/api-gateway/usage#reroute-http-requests) for additional information). 
+
+The following table describes the parameters for `path`:
+
+| Parameter | Description | Type | Required |
+| ---       | ---         | ---  | ---      |
+| `replacePrefixMatch` | Specifies a value that replaces the path prefix for incoming HTTP requests. The operation only affects the path prefix. The rest of the path is unchanged. | String | Required |
+| `type` | Specifies the type of replacement to use for the URL path. You can specify the following values: <ul><li>`ReplacePrefixMatch`: Replaces the the matched prefix of the URL path (default). </li></ul> | String | Optional |
+
+### rules.matches
+
+Specifies rules for matching incoming requests. You can apply [`filters`](#rulesfilters) to requests that match the defined rules. You can match incoming requests based on the following elements:
+
+* [paths](#rules-matches-path) 
+* [headers](#rules-matches-headers) 
+* [query parameters](#rules-matches-queryparams)
+* [request method](#rules-matches-method)
+
+### rules.matches.path
+
+Specifies a list of objects that define matches based on URL path. The following table describes the parameters for the `path` field:
+
+| Parameter | Description | Type | Required |
+| ---       | ---         | ---  | ---      |
+| `type` | Specifies the type of comparison to use for matching the path value. You can specify the following types. <ul><li>`Exact`: Returns a match only when the entire path matches the `value` field (default).</li><li> `PathPrefix`: Returns a match when the path matches the regex defined in the `value` field.</li></ul> | String | Required |
```

Comment:
```suggestion
| `type` | Specifies the type of comparison to use for matching the path value. You can specify the following types. <ul><li>`Exact`: Returns a match only when the entire path matches the `value` field (default).</li><li> `PathPrefix`: Returns a match when the path matches the regex defined in the `value` field.</li></ul> | String | Required |
```
@nathancoleman - Should the `PathPrefix` type actually be `RegularExpression` like the other matching fields? I wasn't sure if this was just an error.